.github/workflows/bc-lint.yml

```diff
@@ -0,0 +1,28 @@
+name: BC Lint
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+    # branches-ignore:
+    #   - <branches to ignore bc-linter>
+    paths:
+      - vllm/attetion/**
+      - vllm/core/**
+      - vllm/v1/attetion/**
+      - vllm/v1/core/**
+
+jobs:
+  bc_lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run BC Lint Action
+        uses: pytorch/test-infra/.github/actions/bc-lint@main
+        with:
+          repo: ${{ github.event.pull_request.head.repo.full_name }}
+          base_sha: ${{ github.event.pull_request.base.sha }}
+          head_sha: ${{ github.event.pull_request.head.sha }}
+          suppression: ${{ contains(github.event.pull_request.labels.*.name, 'suppress-bc-linter') }}
```

Comment:
Just a note that using labels like `suppress-bc-linter` is a common self-serve mechanism in PyTorch PR, but we would need a maintainer to do it here, like adding the `ready` label, which is ok